temporal-serviceclient/src/main/java/io/temporal/serviceclient/ChannelManager.java

```diff
@@ -322,18 +323,43 @@ public HealthCheckResponse healthCheck(
     return stub.check(HealthCheckRequest.newBuilder().setService(healthCheckServiceName).build());
   }
 
-  public Supplier<GetSystemInfoResponse.Capabilities> getServerCapabilities() {
-    return () -> {
-      synchronized (serverCapabilitiesFuture) {
-        GetSystemInfoResponse.Capabilities capabilities = serverCapabilitiesFuture.getNow(null);
+  public Capabilities getServerCapabilities() {
```

Comment:
If  this is public we should not  change the signature.

Reply:
It is not.  It's a package-private class.

Reply:
Ah yeah I saw it being used in other modules, but they are under the same package. My bad